core/ecschema-editing/src/Editing/SchmaEditType.ts

```diff
@@ -0,0 +1,60 @@
+export enum SchemaEditType {
+  CreateSchemaItemFailed = "CreateSchemaItemFailed",
```

Comment:
I'm not sure about this one. Actually, everything in this enum is actually representing an editing operation except of this, which is an editing result (something failed) and therefore a but misleading in this enum I think.

Maybe it make sense to have another enum SchemaEditResultType which derives from SchemaEditType and extend it by the failure type. For the editing calls, we use SchemaEditTypes, in the Errors, the ResultType.
```
export enum SchemaEditResultType = SchemaEditType | {
  CreateSchemaItemFailed = "CreateSchemaItemFailed",
}
```